vividus-plugin-json/build.gradle

```diff
@@ -1,15 +1,13 @@
 project.description = 'VIVIDUS plugin for JSON testing'
 
-ext.jsonPathVersion = '2.8.0'
-
 dependencies {
     implementation project(':vividus-engine')
     implementation project(':vividus-reporter')
     implementation project(':vividus-soft-assert')
     implementation project(':vividus-util')
     api(group: 'net.javacrumbs.json-unit', name: 'json-unit', version: '3.2.2')
-    implementation(group: 'com.jayway.jsonpath', name: 'json-path', version: "${jsonPathVersion}")
-    implementation(group: 'com.jayway.jsonpath', name: 'json-path-assert', version: "${jsonPathVersion}")
+    implementation(group: 'com.github.vividus-framework.JsonPath', name: 'json-path', version: 'fd1bbfe6f9')
+    implementation(group: 'com.jayway.jsonpath', name: 'json-path-assert', version: '2.8.0')
```

Comment:
the versions should be aligned

Reply:
it seems the best way is to move the version to the root `build.gradle`

Reply:
updated, thanks )